tests/vision/vision_conftest.py

```diff
@@ -48,24 +48,24 @@
 
 @pytest.fixture(scope='session')
 def mnist_data_loader_train():
-    return load_mnist_dataset(train=True, object_type='DataLoader')
+    return load_mnist_dataset(train=True, object_type='DataLoader', shuffle=False)
```

Comment:
can't set a seed for shuffling?

Reply:
The shuffling is done inside pytorch, didn't seem like I could.

Reply:
Try this?
https://pytorch.org/docs/stable/notes/randomness.html

Reply:
Thanks